Formula/jid.rb

```diff
@@ -0,0 +1,22 @@
+class Jid < Formula
+  desc "jid"
+  homepage "https://github.com/simeji/jid"
+  version "0.6.2"
+  if Hardware::CPU.is_64_bit?
+    url "#{homepage}/releases/download/#{version}/jid_darwin_amd64.zip"
+    sha256 "3f3cadb4d7842274bd4ed84bc50a8600a655a774aaab73d7a68cf6196dedb4d3"
+  else
+    url "#{homepage}/releases/download/#{version}/jid_darwin_386.zip"
+    sha256 "33b302fb1d4e23036e9acdc9abd0076aff4db8e6eb523bb51a6060c97b3054f4"
+  end
+
+  def install
+    name = Hardware::CPU.is_64_bit? ? "jid_darwin_amd64" : "jid_darwin_386"
+    mv "#{buildpath}/#{name}", "#{buildpath}/jid"
+    bin.install "jid"
+  end
+
+  test do
+    assert_match "jid version v#{version}", shell_output("#{bin}/jid --version")
```

Comment:
No, we'd rather use `assert_match` and `(testpath/"test.json").write` with some sample JSON for a better test.